pkg/operator/ceph/nfs/nfs.go

```diff
@@ -303,12 +303,13 @@ func (r *ReconcileCephNFS) configureNFSPool(n *cephv1.CephNFS) error {
 	poolName := n.Spec.RADOS.Pool
 	logger.Infof("configuring pool %q for nfs", poolName)
 
-	_, err := cephclient.GetPoolDetails(r.context, r.clusterInfo, poolName)
+	args := []string{"osd", "pool", "create", poolName}
```

Comment:
Did you see #10045? Perhaps we should check if the pool exists before creating it?

Reply:
I didn't see that one. After looking at it, I'm not sure it's ideal to only create the pool for certain Ceph versions. I'm inclined to always try to create (with Ceph's defaults) it if it doesn't exist.